src/datasource.test.ts

```diff
@@ -189,10 +190,11 @@ describe('AthenaDatasource', () => {
       expect(queries[0].connectionArgs.region).toEqual('replaced');
     });
   });
+
   it('should not run query if query is empty', async () => {
     const mockedSuperQuery = jest
-      .spyOn(runtime.DataSourceWithBackend.prototype, 'query')
-      .mockImplementation((request: DataQueryRequest<AthenaQuery>) => of());
+      .spyOn(DatasourceWithAsyncBackend.prototype, 'query')
+      .mockImplementation((_: DataQueryRequest<AthenaQuery>) => of({ data: [] }));
```

Comment:
```suggestion
      .mockImplementation(() => of({ data: [] }));
```

Should be able to remove it if it's unused and there's no type errors